### PR TITLE
Add multiple PR templates

### PR DESCRIPTION
**Pull request by fill-the-fill**
Currently we are having a lot of issues with PR including a problem with users not testing build locally before opening a pull request. 

I've modified documentation including adding Checklist before opening a PR and made a suggestion to have multiple PR templates in #742. Seems like the idea was well accepted. I'm attaching a video example of how the PR templates system will look like since that's the only previewing option of my changes:

Please take a look at Files changed and see each PR template individually, I'm open to any suggestions if you think we should change something about the templates.
 
https://user-images.githubusercontent.com/60065019/198557664-d28ca553-4d26-4245-b700-934707fbc93d.mov

### CODE REVIEW

**Review by rphair — APPROVED** [review]
This looked familiar & I realised it was from here (https://github.com/cardano-foundation/developer-portal/issues/742#issuecomment-1287345402); looked good then & looks good now. 🤓

I guess we will want to check ASAP when this is running in production?  I've asked this before (https://github.com/cardano-foundation/developer-portal/pull/832#issuecomment-1285820385) & can I please ask it again (@katomm @rdlrt @fill-the-fill): is there a way to find out when PRs/commits in `staging` are deployed to the production web site?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,100 +1,7 @@
...
-<!--
-
-Leave the ##Quickfix header.  
-
-Don't provide any structure to your description or let the title of the PR speak for itself.
-
-Be advised. PRs with no structure have the lowest priority for maintainers to review / merge and might not make it into the next release of the developer portal.
-
--->
+- [Add Showcase](?expand=1&template=showcase.md)
```
Comment (rdlrt) [inline .github/PULL_REQUEST_TEMPLATE.md:5]: Did we also want to add auto labels (`&label=showcase`)? Thought that was one of the points why we wanted multiple PR templates

**Comment by fill-the-fill** [comment]
> This looked familiar & I realised it was from here ([#742 (comment)](https://github.com/cardano-foundation/developer-portal/issues/742#issuecomment-1287345402)); looked good then & looks good now. 🤓
> 
> I guess we will want to check ASAP after this is running in production? I've asked this before ([#832 (comment)](https://github.com/cardano-foundation/developer-portal/pull/832#issuecomment-1285820385)) & can I please ask it again (@katomm @rdlrt @fill-the-fill): is there a way to find out when PRs/commits in `staging` are deployed to the production web site?

I think we don't have that feature yet, except checking logs with Discord bot, but that would be a good feature. I will look into how we can improve it. =)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (rdlrt) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: yip

Comment (fill-the-fill) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: I can set up auto label bot for the repository in a separate pr that will focus on title and text of the pr. Or would you guys prefer me to do it in this pr as well?

Comment (rdlrt) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: either way is good for me 🙂

Comment (fill-the-fill) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: Am I wrong or I'm seeing some change you've suggested on the PR but I can't tell what exactly do you want to change here

Comment (rdlrt) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: Ye - if going via bot route, no actions needed on this PR 👍🏻

**Review by rdlrt — APPROVED** [review]

**Pull request merged**